glide.yaml

```diff
@@ -11,3 +11,5 @@ import:
   version: v0.10.1
   subpackages:
   - pkg/client/monitoring/v1alpha1
+- package: github.com/robfig/cron
+  version: master
```

Comment:
why specify the version if we are not pinning to any?

Reply:
Only master has the required function.